lib/ansible/modules/clustering/kubevirt/kubevirt_preset.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: kubevirt_preset
+
+short_description: Manage KubeVirt virtual machine presets
+
+description:
+    - Use Openshift Python SDK to manage the state of KubeVirt virtual machine presets.
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+options:
+    state:
+        description:
+            - Create or delete virtual machine presets.
+        default: "present"
+        choices:
+            - present
+            - absent
+        type: str
+    name:
+        description:
+            - Name of the virtual machine preset.
+        required: true
+        type: str
+    namespace:
+        description:
+            - Namespace where the virtual machine preset exists.
+        required: true
+        type: str
+    selector:
+        description:
+            - "Selector is a label query over a set of virtual machine."
+        type: dict
+
+extends_documentation_fragment:
+  - k8s_auth_options
+  - k8s_resource_options
+  - kubevirt_common_options
+
+requirements:
+  - python >= 2.7
+  - openshift >= 0.6.2
+'''
+
+EXAMPLES = '''
+- name: Create virtual machine preset 'myvmir'
+  kubevirt_preset:
+      state: presnet
+      name: myvmir
+      namespace: vms
+      memory: 64M
+      selector:
+        matchLabels:
+            myvmi: myvmi
+
+- name: Remove virtual machine preset 'myvmir'
+  kubevirt_preset:
+      state: absent
+      name: myvmir
+      namespace: vms
+'''
+
+RETURN = '''
+kubevirt_preset:
+  description:
+    - The virtual machine preset managed by the user.
+    - "This dictionary contains all values returned by the KubeVirt API all options
+       are described here U(https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachineinstancepreset)"
+  returned: success
+  type: dict
+'''
+
+import copy
+import traceback
+
+
+from ansible.module_utils.k8s.common import AUTH_ARG_SPEC, COMMON_ARG_SPEC
+
+# FIXME:
```

Comment:
What do we need to fix here?

Reply:
This is hack we use for our unit tests to work.